### DISCUSSION TITLE
The "easy mode" discussion

### DISCUSSION THREAD

**Discussion by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#2110] [General]
I have been hearing about the concept of an easy mode and advance mode for years and I know this type of thing wouldn't work well but it's been talked about on multiple social media platforms so i thought i would make a discussion about this so that we can consolidate the ideals and concepts so it could be more easily understood why this mode would be beneficial or how it wouldn't be

this mode has a lot of concepts that need to be discussed
1. Would inspectors just be gone/hidden?
   - Would this also apply to gizmos?
2. Would the user in easy mode not be able to equip development tools?
   - would this also include any generic tool?
     - would this also include the tooltip multiplexer due to it having the ability to hold development tools?
3. How would interactions work?
   - eg if a user is in easy mode should other users be able to have the ability to inspect that user?
     - this is assuming that the first point is true and would happen because you shouldn't be able to inspect someone if they can't see what you're doing
    
these are just a few examples but I'm free to hear more discussion and ideals about this

**Comment by LexiBasilisk** [comment]
There is nothing to discuss as we are not implementing this, @epicEaston197.

The solution is to continue to incrementally improve the UI/UX of the platform, and its various features, and to provide better reference to existing documentation in-app.

It was previously mentioned as part of #584 that the plan is instead to have "profiles" for user settings, which could also apply to other features like #1749 to allow users to configure their UI layout based on their needs, e.g. socializing, building, streaming, etc.

> On top of these components will be a "routing" system, which routes the appropriate components holding the actual settings to the systems that need them. For each setting group, there might be multiple actual components holding the setting - e.g. some settings can be overridden depending whether the user is in VR or screen.
> 
> The routing system can be more advanced too, adding support for user profiles (e.g. user can switch their settings depending on whether they're socializing, building, streaming or doing anything else) and any other rules.

**Reply by epicEaston197** [reply]
Thank you I mainly made this discussion to consolidate information so that we have a singular reference point if someone brings up "easy mode"

**Comment by Frooxius** [comment]
This has been brought up a number of times, but this isn't something we would do. It ultimately goes against philosophy of how we design Resonite and how Resonite works - it's too much of a black & white solution to a nuanced problem, that will just introduce more problems than it solves.

It doesn't matter how many ideas you'll throw at us for "easy mode", or how much people will yell at us, this is something we fundamentally won't do.

Instead, our approach is to make the initial experience as easy as possible - get user started and be able to get around and socialize. Once they got the basics, they can start exploring more - find more tools, experiences, pieces of UI and so on, to customize their experience along what they want to do and have the platform grow with their individual needs - some people come to just socialize, some come to build, some come to make art or educate - easy/advanced mode isn't good to accommodate this.

If you want to help us make a new user experience easier, then what really helps us the most is helping identify common frustration points for new users. We need to know:
- What does the new user try to accomplish
- What are they trying to do to accomplish it
- What do they get stuck on?

Essentially we need help identifying the "pain points", but not necessarily focus on solutions - that's our job and we can do so in a more comprehensive a holistic manner, that's aligned with the development plans of Resonite.


To give you an example - if we notice user is often stuck wanting to get an avatar - we might want to prioritize stuff like Workshop and such, to make it easy to find already setup avatars and customize those. 

But in this example, the pain point is "I want to get my avatar in. I don't know how. This is what I have." - that's the piece of information we need to know.

**Reply by epicEaston197** [reply]
Yes I agree with this thanks for the information

**Reply by dark-swordsman** [reply]
I think I understand. So the goal is to not remove or obfuscate functionality (what "easy mode" would do), but instead make the functionality easier to understand/use/manage?
